sharktank/sharktank/layers/base.py

```diff
@@ -218,21 +221,7 @@ def export_mlir(self, path: PathLike | None = None, /):
         if path is None:
             raise ValueError("Missing MLIR export path.")
 
-        export_functions = [
-            ExportFunctionConfig(
-                function=self.default_export_function,
-                batch_sizes=self.default_export_batch_sizes,
-            )
-        ]
-        if self.config.export_functions is not None:
-            export_functions = self.config.export_functions
-
-        function_batch_size_pairs = {
-            export_function.function
-            or self.default_export_function: export_function.batch_sizes
-            or self.default_export_batch_sizes
-            for export_function in export_functions
-        }
+        function_batch_size_pairs = self._get_function_batch_sizes_map()
```

Comment:
Nit: IMO "pairs" is misleading because the map is keyed on functions whose values are *lists* of ints. Are you fine with naming it `function_batch_size_map` or `function_batch_size_dict`?

Reply:
I renamed it to `function_batch_sizes_map`.